dev/github/Dockerfile-jdk11

```diff
@@ -153,3 +153,5 @@ RUN ARCH=$(dpkg --print-architecture) && \
     wget --quiet https://releases.hashicorp.com/terraform/1.0.1/terraform_1.0.1_linux_${ARCH}.zip && \
     unzip -o ./terraform_1.0.1_linux_${ARCH}.zip -d /usr/local/bin/ && \
     rm terraform_1.0.1_linux_${ARCH}.zip
+# Rust toolchains for native components
+RUN apt-get install -y rust-all
```

Comment:
append `rust-all` to the `apt-get install` on line 141. reduces the number of image layers that are cached. similar for other dockerfiles

Reply:
Ok. But let me remove this change from this PR and move them into a dedicated PR that updates the docker images.

Reply:
sgtm